jina/flow/base.py

```diff
@@ -169,7 +172,7 @@ def _add_gateway(self, needs, **kwargs):
                 name=pod_name,
                 ctrl_with_ipc=True,  # otherwise ctrl port would be conflicted
                 runtime_cls='GRPCRuntime'
-                if self._cls_client == Client
+                if self._cls_client == GrpcClient
```

Comment:
better uniform the case, `GRPCRuntime` then `GRPCClient` or `GrpcRuntime` and `GrpcClient`